src/test/groovy/ee/tuleva/onboarding/auth/ocsp/OCSPRequestSpec.groovy

```diff
@@ -0,0 +1,183 @@
+package ee.tuleva.onboarding.auth.ocsp
+
+
+import spock.lang.Specification
+
+import java.security.cert.X509Certificate
+
+class OCSPRequestSpec extends Specification {
+
+    def "Test if certificate is valid for (TEST of ESTEID2018.pem.cer)"() {
```

Comment:
Are these tests running against the real OCSP service? We should mock it. 
Tests should not be dependent on external resources.